src/Ilios/CoreBundle/Form/Type/UserType.php

```diff
@@ -88,6 +88,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'required' => false,
                 'entityName' => "IliosCoreBundle:Cohort"
             ])
+            ->add('permissions', 'tdn_many_related', [
```

Comment:
Remove this and don't allow a save on the many side of a many to one.  We can add it back in later.